part-spec/common/thermals.json

```diff
@@ -0,0 +1,81 @@
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "ambientTemp": {
+                "description": "recommended operating ambient air temperature range of a part",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "ambientTempMax": {
+                "description": "absolute maximum ambient air temperature range under which a part operates",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "caseTemp": {
+                "description": "case temperature range of a part",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "caseTempMax": {
+                "description": "absolute maximum case temperature range of a part",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "leadTempMax": {
+                "description": "absolute maximum lead temperature of a part",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "storageTempMax": {
+                "description": "absolute maximum storage temperature range of a part",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "packageThermalResistance": {
+                "description": "package thermal resistance of a part",
+                "comment": "units of Celsius/watt or kelvin/watt",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "ThermalResistanceJunctionToAmbient": {
+                "description": "thermal resistance between the semiconductor junction of a part and the ambient air",
+                "comment": "units of Celsius/watt or kelvin/watt",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "ThermalResistanceJunctionToCase": {
+                "description": "thermal resistance between the semiconductor junction and the package surface of a part",
+                "comment": "units of Celsius/watt or kelvin/watt",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "ThermalResistanceJunctionToboard": {
+                "description": "thermal resistance between the semiconduction junction of a part and the board",
+                "comment": "units of Celsius/watt or kelvin/watt",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "ThermalResistanceJunctionToLead": {
+                "description": "thermal resistance between the semiconduction junction of a part and the soldering point",
```

Comment:
how about: "...junction of a part and the solder pad"

Reply:
changed